src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -587,6 +587,11 @@ public bool IsBindableToken(SyntaxToken token)
                 return true;
             }
 
+            if (token.IsKind(SyntaxKind.EqualsGreaterThanToken))
```

Comment:
I did something comparable while adding support for QuickInfo for Linq query syntax [here](https://github.com/dotnet/roslyn/pull/23049/files#diff-1eda1d67f1b7acf2ea04b9af6d0e105aR594). I was wondering whether this might have unwanted side effects (In my case it is even worse, because it is a CommaToken). If you experience something unexpected it would be nice to get me informed.

Reply:
@jason@jason-m.com Can you clarify the concern?
The method is only used in `GetSemanticInfoAtPositionAsync` which now supports finding a symbol when the position is in `=>`.
`GetSemanticInfoAtPositionAsync` itself is used in `GoToDefinition`, `Peek` and `SymbolFinder` (which is used in numerous places, like `FindUsages`, `FindReferences`, ...).

---
In reply to: [161932149](https://github.com/dotnet/roslyn/pull/23970#discussion_r161932149) [](ancestors = 161932149)